### PR TITLE
sci-libs/matio: Version bump + bump to EAPI=6

### PR DESCRIPTION
**Pull request by marbre**
This version of matio fixed a whole bunch of bugs. I further removed autotools-utils which isn't needed. Beside libtool was added to the build deps.

### CODE REVIEW

**Comment by kensington** [comment]
@gentoo/sci

File: sci-libs/matio/matio-1.5.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+RDEPEND="
+	sys-libs/zlib
+	hdf5? ( sci-libs/hdf5 )"
+DEPEND="${RDEPEND}
+	sys-devel/libtool
+	doc? ( virtual/latex-base )"
+
+src_configure() {
+	econf \
+		-docdir="${EPREFIX}/usr/share/doc/${PF}" \
```
Comment (SoapGentoo) [inline sci-libs/matio/matio-1.5.7.ebuild]: not needed anymore with `EAPI=6`

File: sci-libs/matio/matio-1.5.7.ebuild [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+		$(use_enable hdf5 mat73) \
+		$(use_enable sparse extended-sparse)
+}
+
+src_compile() {
+	emake
+	use doc && 	emake -C documentation pdf
```
Comment (SoapGentoo) [inline sci-libs/matio/matio-1.5.7.ebuild]: what is the weird space doing in there, is this maybe a tab?

File: sci-libs/matio/matio-1.5.7.ebuild [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+src_configure() {
+	econf \
+		-docdir="${EPREFIX}/usr/share/doc/${PF}" \
+		$(use_enable hdf5 mat73) \
+		$(use_enable sparse extended-sparse)
+}
+
+src_compile() {
+	emake
```
Comment (SoapGentoo) [inline sci-libs/matio/matio-1.5.7.ebuild]: switch this to `default` instead

File: sci-libs/matio/matio-1.5.7.ebuild [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+	use doc && 	emake -C documentation pdf
+}
+
+src_install() {
+	emake DESTDIR="${D}" install
```
Comment (SoapGentoo) [inline sci-libs/matio/matio-1.5.7.ebuild]: `default` here too

File: sci-libs/matio/matio-1.5.7.ebuild (hunk 2 above)
Comment (marbre) [inline sci-libs/matio/matio-1.5.7.ebuild]: It's a **take over** from the lower versions. Will remove it in the revised ebuild.

**Comment by marbre** [comment]
@SoapGentoo All done.

**Comment by SoapGentoo** [comment]
Thanks @marbre for the PR, your work is always appreciated :smile:

**Pull request closed without merging**